src/scripts/experience/topic.js

```diff
@@ -178,9 +182,9 @@ define([
             issue.tapS.remove(issue.topicTap);
         }
 
-        for(i = 0; i < this._fakes.length; i ++) {
-            this._fakes[i].explode();
-        }
+        // for(i = 0; i < this._fakes.length; i ++) {
+        //     this._fakes[i].explode();
+        // }
```

Comment:
If the above is not required, you can just delete it, no need to comment it out.